src/main/java/com/github/junrar/LocalFolderExtractor.java

```diff
@@ -50,6 +50,9 @@ File extract(
         final File f = createFile(fileHeader, folderDestination);
         try (OutputStream stream = new FileOutputStream(f)) {
             arch.extractFile(fileHeader, stream);
+        } catch (Exception e) {
+            System.out.println(e);
```

Comment:
you shouldn't use standard streams, use the Logger instead

Reply:
yeah,i'll fix it.
This is the code for debugging, I forgot to delete it